scripts.js

```diff
@@ -0,0 +1,38 @@
+(function () {
+  console.log('hey');
```

Comment:
Console.log is normally used for testing / development.  Likely you do not want to write to the console unless you have very specific reasons to in the finished code.